src/DatePicker/DatePicker.js

```diff
@@ -200,10 +232,18 @@ class DatePicker extends Component {
     this.openDialog();
   }
 
+  shouldHandleKeyboard() {
+    return this.props.keyboardEnabled && !this.props.disabled &&
+      !this.props.locale && !this.props.shouldDisableDate &&
+      !this.isControlled();
```

Comment:
I'm worried about this feature being disabled for controlled component.
On the `next` branch, we only focus on stateless component, hence, they are all controlled at their lowest implementation level.
What's the issue?